apis/kueue/v1alpha1/cohort_types.go

```diff
@@ -64,8 +64,15 @@ type CohortSpec struct {
 	FairSharing *kueuebeta.FairSharing `json:"fairSharing,omitempty"`
 }
 
+type CohortStatus struct {
+	// FairSharing contains the information about the current status of fair sharing.
+	// +optional
```

Comment:
delete please; then the doc will be generated using v1beta1.FairSharing's comment, I believe

Reply:
Done